tests/bin/check/run.t

```diff
@@ -0,0 +1,194 @@
+A necessary condition for dune-release to work is that it's able to infer the main package
+
+    $ touch first_pkg.opam
+    $ touch other_pkg.opam
+
+    $ dune-release check --working-tree
+    [FAIL] cannot determine distribution name automatically: add (name <name>) to dune-project
+    [1]
+
+Let's clean up to not interfere with the following tests
+
+    $ rm first_pkg.opam other_pkg.opam
+
+If the condition described above is fulfilled, there are 4 checks to be performed
+
+    $ cat > my_pkg.opam <<EOF \
```

Comment:
Wouldn't it make more sense to move this part right after the `rm` line above?

Reply:
Ok, I've separated each test set up from its test in general now.